client/reader/discover/discover-stream.js

```diff
@@ -49,6 +49,12 @@ const DiscoverStream = ( props ) => {
 		},
 	} );
 
+	// Add dailyprompt to the front of interestTags if not present.
+	const hasDailyPrompt = interestTags.filter( ( tag ) => tag.slug === 'dailyprompt' ).length;
+	if ( ! hasDailyPrompt ) {
+		interestTags.unshift( { title: translate( 'Daily Prompt' ), slug: 'dailyprompt' } );
```

Comment:
Can we use `Daily prompts` as the title to align with `First posts`?
```suggestion
		interestTags.unshift( { title: translate( 'Daily prompts' ), slug: 'dailyprompt' } );
```

Reply:
If we want parity with the tag title, it looks like we would actually use "Dailyprompt" as a single word. But given the context of this button, I think your suggestion makes sense. Committing your suggestion.

<img width="777" alt="Screenshot 2023-10-27 at 11 32 09 AM" src="https://github.com/Automattic/wp-calypso/assets/28742426/748f8536-2801-45f2-b52f-afc119eb055f">